src/Host.cpp

```diff
@@ -377,6 +377,7 @@ Host::Host(int port, const QString& hostname, const QString& login, const QStrin
 , mPlayerRoomInnerColor(Qt::white)
 , mPlayerRoomOuterDiameterPercentage(120)
 , mPlayerRoomInnerDiameterPercentage(70)
+, mProfileStyleSheet(QString(""))
```

Comment:
To specify an empty `QString` you ought to use `QString()` the difference between an empty `QString` and a null one is pretty much only distinguishable via `QString::isNull()` and actually specifying an empty `QString` here is (almost?) pointless. :stuck_out_tongue_winking_eye: 

:information_source: As it happen you do not need to initialise `Host::mProfileStyleSheet` in this manner - the default constructor (which is what will be used anyhow if you do not provide an initialiser) will make it an empty (well null) string anyhow...

Reply:
Thanks :) didn't know that.
Will change it.